### PR TITLE
add blog for lambda releases

### PR DESCRIPTION
**Pull request by cartersocha**
Adding a blog post from the FAAS sig

### CODE REVIEW

**Review by svrnm — APPROVED** [review]
lgtm

File: content/en/blog/2023/lambda-release.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+If you've been monitoring Lambdas using OTel for a while now you may be slightly
+confused by this announcement. You might say something like OTel has had a
+[repo for Lambda layers](https://github.com/open-telemetry/opentelemetry-lambda)
+and they've been available on AWS for years.
+
+Which is totally correct. Thankfully, we're not remaking the wheel here.
+However, there are a couple new updates.
+
+Previously the OTel Lambda layers were only released as part of the Amazon
+Distribution of OTel (ADOT) and the community had limited control over releases,
```
Comment (svrnm) [inline content/en/blog/2023/lambda-release.md:24]:
```suggestion
Previously the OTel Lambda layers were only released as part of the 
[Amazon Distribution of OTel (ADOT)](https://aws-otel.github.io/)
and the community had limited control over releases,
```

**Review by tylerbenson — APPROVED** [review]
Looks great!

File: content/en/blog/2023/lambda-release.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+If you've been monitoring Lambdas using OTel for a while now you may be slightly
+confused by this announcement. You might say something like OTel has had a
+[repo for Lambda layers](https://github.com/open-telemetry/opentelemetry-lambda)
+and they've been available on AWS for years.
+
+Which is totally correct. Thankfully, we're not remaking the wheel here.
+However, there are a couple new updates.
+
+Previously the OTel Lambda layers were only released as part of the
```
Comment (codeboten) [inline content/en/blog/2023/lambda-release.md:23]: I would split the things in this paragraph into a list to make it more clear what's changing and why

Comment (cartersocha) [reply content/en/blog/2023/lambda-release.md]: Updated and lmk what you think.

**Review by codeboten — APPROVED** [review]
Looks good, thanks @cartersocha

File: content/en/blog/2023/lambda-release.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+If you've been monitoring Lambdas using OTel for a while now you may be slightly
+confused by this announcement. You might say something like OTel has had a
+[repo for Lambda layers](https://github.com/open-telemetry/opentelemetry-lambda)
+and they've been available on AWS for years.
+
+Which is totally correct. Thankfully, we're not remaking the wheel here.
+However, there are some pre-existing problems that impact our users:
+
+- The OTel Lambda layers were only released as part of the
+  [Amazon Distribution of OTel (ADOT)](https://aws-otel.github.io/) and the
```
Comment (mhausenblas) [inline content/en/blog/2023/lambda-release.md:24]: It's AWS Distro for OpenTelemetry (ADOT)

**Review by mhausenblas — DISMISSED** [review]

File: content/en/blog/2023/lambda-release.md (hunk 3 above)
Comment (cartersocha) [reply content/en/blog/2023/lambda-release.md:24]: updated - thanks

**Comment by cartersocha** [comment]
@svrnm  / @cartermp - should be good to go whenever

**Review by cartermp — APPROVED** [review]
LGTM as well, @svrnm let's figure out the publish date, then we can merge this pupperoo

**Comment by svrnm** [comment]
> LGTM as well, @svrnm let's figure out the publish date, then we can merge this pupperoo

Lets close the histogram series with #2661 today, keep it on top of the blog for a few days and then let's merge in this one, maybe Thursday?

File: content/en/blog/2023/lambda-release.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+date: 2023-05-15
+author: '[Carter Socha](https://github.com/cartersocha) (Lightstep)'
+---
+
+The Functions-as-a-Service (FAAS) SIG is incredibly excited to announce that the
+OpenTelemetry (OTel) community is now releasing our own Lambda layers and has
+[centralized documentation](https://opentelemetry.io/docs/faas/) on how to
+monitor Amazon Web Services (AWS) Lambdas.
+
+### A Slightly New World
```
Comment (chalin) [inline content/en/blog/2023/lambda-release.md:13]: The heading level should be at level 2.

```suggestion
## A Slightly New World
```

I'm not sure how much meaning that title text conveys. How about:

```suggestion
## OTel FaaS repackaged
```

File: content/en/blog/2023/lambda-release.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
+---
+title: OpenTelemetry Updates our Lambda Support
```
Comment (chalin) [inline content/en/blog/2023/lambda-release.md:2]:
```suggestion
title: OpenTelemetry Updates Lambda Support
```

File: content/en/blog/2023/lambda-release.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+linkTitle: OTel Lambda Updates
+date: 2023-05-15
```
Comment (chalin) [inline content/en/blog/2023/lambda-release.md:4]:
```suggestion
date: 2023-05-25
```

File: content/en/blog/2023/lambda-release.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+author: '[Carter Socha](https://github.com/cartersocha) (Lightstep)'
+---
+
+The Functions-as-a-Service (FAAS) SIG is incredibly excited to announce that the
```
Comment (chalin) [inline content/en/blog/2023/lambda-release.md:8]: Fix capitalization. Also, link to a FaaS resource? E.g.:

```suggestion
The [Functions-as-a-Service (FaaS) SIG](https://docs.google.com/document/d/187XYoQcXQ9JxS_5v2wvZ0NEysaJ02xoOYNXj08pT0zc) is incredibly excited to announce that the
```

File: content/en/blog/2023/lambda-release.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+  Java, and Python.
+- Added official community Lambda documentation to the OTel website under the
+  new FAAS section.
+
+### Get Involved
+
+We're not stopping at our current state. Going forward, the FAAS SIG is planning
+on adding similar documentation + (potentially) auto-instrumentation for other
+Cloud vendors like Azure and GCP, enhancing the existing Lambda assets, and
+improving OpenTelemetry performance for Function specific scenarios.
```
Comment (chalin) [inline content/en/blog/2023/lambda-release.md:49]:
```suggestion
Moving forward, the FaaS SIG plans to: enhance the documentation, add auto-instrumentation for other
Cloud vendors like Azure and GCP (tentative), enhancing the existing Lambda assets, and
improving OpenTelemetry performance for Function specific scenarios.
```

**Review by chalin — COMMENTED** [review]
Copyedits

**Comment by cartersocha** [comment]
Thanks @chalin

**Comment by chalin** [comment]
I just rebased. That should fix the link-check failures.

**Comment by chalin** [comment]
Oh, no. This PR has the wrong version of the spec. Let me fix that ...

**Comment by chalin** [comment]
Somehow, 7c16f7614bc4022b57c7ca23e8ae1fdfab543243 pulls in an old version of the spec. I've partially reverted it.

**Review by chalin — APPROVED** [review]
👍

**Comment by svrnm** [comment]
Live! https://opentelemetry.io/blog/2023/lambda-release/

**Pull request merged**